packages/botonic-nlp/src/tasks/ner/storage/config-storage.ts

```diff
@@ -0,0 +1,19 @@
+import { mkdirSync, readFileSync, writeFileSync } from 'fs'
+import { join } from 'path'
+
+import { CONFIG_FILENAME } from '../../../storage/constants'
+import { NerConfig } from './types'
+
+export class NerConfigStorage {
+  static load(path: string): NerConfig {
+    const config = JSON.parse(
+      readFileSync(join(path, CONFIG_FILENAME), 'utf-8')
```

Comment:
not sure if there is a native object called encodings for utf8, but this should be a constant, not a literal